### ISSUE TITLE
Minimise button doesn't work on Mac

### ISSUE THREAD

**Issue by codykonior** [rmcrackan/Libation#1037]
**Describe the bug**
On Mac there's the traffic lights in the top left for close, minimize, or full screen.

The minimize (yellow) button doesn't seem to do anything. Ideally it should minimize the main window to the Dock.

It's not world-ending but :-) 

**Screenshots**
<img width="420" alt="Screenshot 2024-11-12 at 3 21 00 pm" src="https://github.com/user-attachments/assets/f3395232-95a7-4e2b-8843-c87b1e43ad8a">

**Platform**

Mac Sequoia. Libation is v11.1.0.1.

**Comment by tothemoonsands** [comment]
I opened a similar issue (#983) and I found that if you click the minimize button (or hide via long click on dock icon), then the app eventually crashes. The only way you can keep it open long-term for periodic scanning is by keeping the window active/in view. 

I think the dev is aware of this issue, but unfortunately no fix yet.

**Comment by rmcrackan** [comment]
> I think the dev is aware of this issue, but unfortunately no fix yet.

Correct. I don't even own a mac. I'm leaving these open in hopes that one day I'll find another developer who can help with mac.

**Comment by rmcrackan** [comment]
@Mbucari Can you do a quick check to see if this button works for the latest build on your mac?

**Comment by codykonior** [comment]
It works for me on 12.0.2.2. Yay!

**Comment by rmcrackan** [comment]
w00h00 -- thanks for the update :)
